src/engine/lua.cc

```diff
@@ -297,7 +297,7 @@ int Lua::getvars(lua_State *L) {
     variables::Variable::stringMatchResolveMulti(t, varname, &l);
 
     lua_newtable(L);
-    for (auto i : l) {
+    for (auto* i : l) {
```

Comment:
```suggestion
    for (const auto* i : l) {
```
The [original](https://github.com/owasp-modsecurity/ModSecurity/commit/a676f313c3f9277ae007b09d79fa698f61947ce6) code was written about 8 years ago, the author is no longer involved in development.

If I had to guess why they used value copy mechanism, I would say they wanted to avoid any chance to modify the value on Lua stack (except for intentional change with `setvar()`).

If we want to keep this restriction, please consider to use `const` modifier here.